BrickBreaker.js

```diff
@@ -95,6 +94,10 @@ class Game{
 	}
 
 	drawObjects(){
+		$("#start").click(function() {
+			this.draw(context);
+		});
```

Comment:
```suggestion
                this.draw(this.canvas)
```
draw는 언제 그려져야 할까요? 아래에 `clearRect`는 캔버스 전체를 지웁니다.

Reply:
context라는 변수를 만들지 않았어요. `game.canvas`라는 변수를 통해 캔버스에 접근할 수 있습니다..